src/main/java/io/debezium/connector/yugabytedb/YugabyteDBSnapshotChangeEventSource.java

```diff
@@ -603,14 +603,6 @@ private OpId getTransactionStartLsn() throws SQLException {
     @Override
     protected void complete(SnapshotContext<YBPartition, YugabyteDBOffsetContext> snapshotContext) {
         snapshotter.snapshotCompleted();
-
```

Comment:
Can you help me understand this removal? The connections should not be closed after snapshot because the same connections are used in streaming?

Reply:
This removal is more from the test POV, some of the tests seemed to be failing intermittently saying that the client connection was closed in this function.

Then on investigation, I realised this `complete()` function is to just mark the snapshot as completed, the closing of `YBClient` instances should be done at the very end of the connector i.e. when it is killed or stopped - and we have a function doing the same already in `YugabyteDBConnector` so the reason why I was getting the exception was that I was simply trying to stop a already stopped instance of the client instance.